src/profile-logic/marker-schema.js

```diff
@@ -0,0 +1,217 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+import type { MarkerFormatType } from 'firefox-profiler/types';
+import {
+  formatNumber,
+  formatPercent,
+  formatBytes,
+  formatSeconds,
+  formatMilliseconds,
+  formatTimestamp,
+  formatMicroseconds,
+  formatNanoseconds,
+} from '../utils/format-numbers';
```

Comment:
Super nit: We usually do normal imports first, and then the type imports. It would be great to do that here too :)